### PR TITLE
Update CORTX_on_Open_Virtual_Appliance.rst

### PR DESCRIPTION
**Pull request by daniarherikurniawan**
### Changes
 - [v] Why is this change required? 
 I successfully tested this tutorial.

### Checklist
 - [v] tested locally

### CODE REVIEW

File: doc/OVA/1.0.4/CORTX_on_Open_Virtual_Appliance.rst [hunk 1]
```diff
@@ -310,6 +310,8 @@ Restart CORTX
    
 Tested by:
 
+- June 20, 2021: Daniar Kurniawan (daniar@uchicago.edu) using OVA release 1.0.3 on MacOS Catalina running Oracle VirtualBox 5.2.18 (Qt5.6.3).
+
```
Comment (mukul-seagate11) [inline doc/OVA/1.0.4/CORTX_on_Open_Virtual_Appliance.rst:314]: Daniar, this wiki is for 1.0.4 as per newer OVA version published yesterday

Comment (daniarherikurniawan) [reply doc/OVA/1.0.4/CORTX_on_Open_Virtual_Appliance.rst:314]: Yes, I am testing the 1.04 currently. I will update this PR soon.

Comment (johnbent) [reply doc/OVA/1.0.4/CORTX_on_Open_Virtual_Appliance.rst:314]: What is status of this PR please?

Comment (daniarherikurniawan) [reply doc/OVA/1.0.4/CORTX_on_Open_Virtual_Appliance.rst:314]: It turns out that there is a problem with the newest version (1.0.4). I am stuck on `starting consul server`. I have been redoing all the steps and also trying to use rescue-OS and non rescue-OS, but so far the result is still the same. I am still investigating the problem. I will file an issue if the problem still exists.

Comment (johnbent) [reply doc/OVA/1.0.4/CORTX_on_Open_Virtual_Appliance.rst:314]: @daniarherikurniawan , do we still need this one?

Comment (daniarherikurniawan) [reply doc/OVA/1.0.4/CORTX_on_Open_Virtual_Appliance.rst:314]: Hi @johnbent, Thanks for the reminder. I will close this for now. I haven't been able to run the newest OVA; I am still focusing on the S3 server.

**Pull request closed without merging**